2-With-broker/README.md

```diff
@@ -124,25 +124,21 @@ where `[APPLICATIONID]` is the identifier you copied in step 2. Save the file.
 
 #### [OPTIONAL] Step 3b: Configure the Android project with your return URI
 
-1. Open the `UserDetailsClient.Droid\MainActivity.cs` file.
-2. Open the `UserDetailsClient.Droid\Properties\AndroidManifest.xml`
-3. Add or modify the `<application>` element as in the following:
-
-```Xml
-    <application>
-    <activity android:name="microsoft.identity.client.BrowserTabActivity">
-      <intent-filter>
-    <action android:name="android.intent.action.VIEW" />
-    <category android:name="android.intent.category.DEFAULT" />
-    <category android:name="android.intent.category.BROWSABLE" />
-    <data android:scheme="msal[APPLICATIONID]" android:host="auth" />
-      </intent-filter>
-    </activity>
-      </application>
+1. Open the `UserDetailsClient.Droid\MsalActivity.cs` file.
+1. Replace `[ClientID]` with the identifier you copied in step 2.
+1. Save the file.
+
+```csharp
+  [Activity]
+  [IntentFilter(new[] { Intent.ActionView },
```

Comment:
I think that what we should do is inherit from `IntentFilter` and add a constructor with the client ID, which would compute the DataScheme ?

Reply:
@jmprieur `IntentFilter` is a sealed class.